multinode/bootstrapping.md

```diff
@@ -0,0 +1,191 @@
+# Data Node Bootstrapping [](distdb-bootstrap)
+
+Scaling up multi-node Timescale is achieved by adding data nodes to
+the access nodes using the `add_data_node` and `attach_data_node`
+functions. In order for a data node to be available to the access node
+it is necessary to ensure that it is properly setup to work as a data
+node. This process is refereed to as *bootstrapping the data node*.
+
+## Connections to the data node
+
+All connections to the data node are made as the current user on the
+access node. Which requires that:
+
+1. The current user exists on both the data node and access node.
+2. The data node is set up to handle logins using either certificates
+   or password files, as described in the [Data Node Authentication
+   Chapter][distdb-auth].
+
+## Adding data nodes to the access node
+
+When an access node is adding a new data node to the distributed
+database it either bootstraps the data node itself or verifies that
+the data node is properly bootstrapped.
+
+A new data node is added to the access node using the `add_data_node`
+command.
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com');
+```
+
+The database to bootstrap on the data node is given by the name of the
+current database on the access node. For example, if you are connected
+to the `my_data` database on the access node, the database name will
+be `my_data` on the data node as well.
+
+If you do not want to use the same database on the data node as on the
+access node, this can be overridden using the `database` option to the
+`add_data_node` command.
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com',
+                            database => 'other');
+```
+
+The access node has to ensure that there is a properly configured
+database on the data node by either bootstrapping a new database or
```

Comment:
IMHO it is important to mention that the above examples with omitted `bootstrap` option means `true`.